CHANGELOG.md

```diff
@@ -7,8 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
 - Support warning about and automatically fixing some simple uses of some 1.0 elements to their 2.0 usage styles.
+  - To add this fixable lint warning for your own elements, simply add the `old-content-selector` attribute to your new `<slot>` element. Set its value to the value that your old `<content>` element had for its `select` attribute.
```

Comment:
I'd add an example here, wdyt of something like
```html
<!-- <content select=".some-content"></content> -->
<slot old-content-selector=".some-content"></slot>

Reply:
+1, wrote it out more explicitly